sequencer/txselector/txselector.go

```diff
@@ -123,3 +124,224 @@ func (txsel *TxSelector) LocalAccountsDB() *statedb.LocalStateDB {
 func (txsel *TxSelector) Reset(batchNum common.BatchNum, fromSynchronizer bool) error {
 	return common.Wrap(txsel.localAccountsDB.Reset(batchNum, fromSynchronizer))
 }
+
+// GetL1L2TxSelection returns the selection of L1 + L2 txs.
+// It returns: the CoordinatorIdxs used to receive the fees of the selected
+// L2Txs. An array of bytearrays with the signatures of the
+// AccountCreationAuthorization of the accounts of the users created by the
+// Coordinator with L1CoordinatorTxs of those accounts that does not exist yet
+// but there is a transactions to them and the authorization of account
+// creation exists. The L1UserTxs, L1CoordinatorTxs, PoolL2Txs that will be
+// included in the next batch.
+func (txsel *TxSelector) GetL1TxSelection(selectionConfig txprocessor.Config,
+	l1UserTxs, l1UserFutureTxs []common.L1Tx) ([][]byte, []common.L1Tx, error) {
+	accCreationAuths, l1UserTxs, err := txsel.getL1TxSelection(selectionConfig, l1UserTxs, l1UserFutureTxs)
+	return accCreationAuths, l1UserTxs, err
+}
```

Comment:
I'm not sure why we need this function and txSelector, as you can see this function doesn't do anything. [[ref](https://www.notion.so/tokamak/TxSelector-WIP-15bd96a400a3808d977de31b19c97210)]

Reply:
You are right, txSelection is not doing anything for now. But I am not sure if we totally want to remove tx selection ? or we will need to select L1 transactions with use of txSelection and need to update it ?

Reply:
Sure